GVFS/GVFS.Installer.Mac/CreateMacInstaller.sh

```diff
@@ -1,18 +1,24 @@
 #!/bin/bash
 
-CONFIGURATION=$1
+SOURCEDIRECTORY=$1
+if [ -z $SOURCEDIRECTORY ]; then
+  echo "Build configuration not specified"
```

Comment:
Change `"Build configuration not specified"` to `"Source directory not specified"`